bot_control.py

```
@@ -0,0 +1,110 @@
+from typing import Dict, Optional, List
+import discord
+
+class BotControl:
```

Flake8 (7.2.0)

[error] expected 2 blank lines, found 1

(E302)